sympy/physics/vector/tests/test_point.py

```diff
@@ -216,7 +216,10 @@ def test_auto_point_vel_shortest_path():
     O1 = Point('O1')
     O1.set_pos(O, q2 * B.z)
     P4.set_pos(O1, q1 * B.x + q2 * B.z)
-    assert P4.vel(B) == q1.diff(t) * B.x + u2 * B.y + 2 * q2.diff(t) * B.z
+    with warnings.catch_warnings():
+        warnings.simplefilter('error')
+        with ignore_warnings(UserWarning):
+            assert P4.vel(B) == q1.diff(t) * B.x + u2 * B.y + 2 * q2.diff(t) * B.z
```

Comment:
It'd be helpful to have a comment explaining what is happening in each warning catch, as it isn't obvious from a quick read of the code. Does this one now cause a warning? Which one the cyclic or multipath one?

Reply:
multiplath warning

Reply:
Added comment

Reply:
So here it is supposed to get that warning? It isn't clear what each warning catch (in each test is doing). You should write a comment just above `with` in each one that is something like "there are two possible paths in this point tree, thus a warning is raised". Should you also ensure that the correct warning is raised, as there are two possible warnings.

Reply:
Added comments and also ensured that correct warning is being raised

Reply:
Sorry about requesting that, I missed that the names of the functions explained what was going on too. I thought they were all old tests you added warning catching too. Regardless, it is clearer now. Thanks.